### DISCUSSION TITLE
creat repo link for Russia termux

### DISCUSSION THREAD

**Discussion by ExtinctDrift19** [termux/termux-packages#9549] [Ideas]
increase internet speed in russia link repository Russian yandex yandex cloud

**Comment by ghost** [comment]
Someone or you should setup an APT mirror repository for Termux for people who live in Russia

https://github.com/termux/termux-packages/wiki/How-to-mirror-the-official-repositories

**Comment by Maxython** [comment]
> The idea is interesting, but impossible to implement. And the reason here is not because of the situation in the world, but because of the very idea of ​​keeping apt packages in a Yandex disk. 
>
> I explain: 
> - Firstly, Yandex disk allows you to use 10 GB for free, but for storing termux packages this is not enough, in addition to this, the amount of storage will increase. 
> - Secondly, the Yandex disk service is intended for personal use, and not for programming.
>
> I would recommend you to find a Russian service for storage, something similar to aws bucket.

I'm sorry, I didn't read what service you are going to use.

**Reply by AKPR2007** [reply]
> The idea is interesting, but impossible to implement. And the reason here is not because of the situation in the world, but because of the very idea of ​​keeping apt packages in a Yandex disk.
> 
> I explain:
> 
> * Firstly, Yandex disk allows you to use 10 GB for free, but for storing termux packages this is not enough, in addition to this, the amount of storage will increase.
> * Secondly, the Yandex disk service is intended for personal use, and not for programming.
> 
> I would recommend you to find a Russian service for storage, something similar to aws bucket.

I was thinking of setting up a cloud mirror on gdrive / onedrive . How much storage is estimated ? I have 180gb free

**Reply by ghost** [reply]
Is Google Drive/Onedrive supports direct links to files and directories? those cloud content providers are not suitable for hosting APT repo and just meant for storing files. and Google or MS might kill us from using much traffic, another issue is syncing apt repo from packages.termux.org to OneDrive/MS, it's a tedious task for syncing apt repo manually hand by hand and you cannot use aptly or rsync to automatically sync the mirror from the main because those cloud storage are backed by the companies

**Comment by ghost** [comment]
@ExtinctDrift19 Please post your links in one comments than spamming individual links per comment

**Comment by Exanoff12** [comment]
i mean. you can use the mirror hosted in Ukraine 😬 at `termux.astra.in.ua` (see documentation for more information). it will not give you the speed as much as it would if hosted on Russia. but still will be better than other ones.

i am surprised on how they still kept the service running in this almost ww3 situation.

**Comment by ExtinctDrift19** [comment]
site URL Russian tutorials cloud.yandex.ru 
word search yandex repository create

**Comment by 2096779623** [comment]
https://github.com/termux/termux-packages/wiki/Mirrors#mirrors-hosted-in-russia
